<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startUI</class>
 <widget class="QWidget" name="startUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>674</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>方正启体简体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>OpenCV Face Recognition System</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>481</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正启体简体</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>基与OpenCV的人脸识别系统</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="functionBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1121</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正启体简体</family>
    </font>
   </property>
   <property name="title">
    <string>OpenCV Face Recognition System</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正启体简体</family>
     </font>
    </property>
    <property name="title">
     <string>主要功能</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>点击签到</string>
     </property>
     <widget class="QPushButton" name="signButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QpushBuuton {
border:none;
color:white;
}
</string>
      </property>
      <property name="text">
       <string>点击签到</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>录入数据</string>
     </property>
     <widget class="QPushButton" name="faceRecordButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>录入数据</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>数据管理</string>
     </property>
     <widget class="QPushButton" name="dataManageButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>数据管理</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>351</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正启体简体</family>
     </font>
    </property>
    <property name="title">
     <string>帮助与支持</string>
    </property>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>19</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>更多信息</string>
     </property>
     <widget class="QPushButton" name="githubButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>GitHub仓库</string>
      </property>
     </widget>
     <widget class="QPushButton" name="developerButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>联系开发者</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>90</y>
       <width>301</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>打赏作者</string>
     </property>
     <widget class="QLabel" name="wechatLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>175</width>
        <height>175</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>查看签到情况</string>
    </property>
    <widget class="QPushButton" name="flushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>点我刷新</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;签到成功后刷新查看&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="signBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>551</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正启体简体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>                                                                                   签到数据</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>541</width>
      <height>531</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <column>
     <property name="text">
      <string>学号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>班级</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>是否签到</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>签到时间</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="helpBox">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>190</y>
     <width>181</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正启体简体</family>
    </font>
   </property>
   <property name="title">
    <string>使用帮助</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>   1. 第一次使用本系统时，需录入数据，然后在数据管理界面训练人脸数据，最后签到。
  2.录入数据时，可以同时多人先后录入。
  3.数据管理可以进行删除操作，删除数据后，需要重新训练数据。
  4.若要实现早晚签到，则需要早晚各运行一次本软件。</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
